eox_nelp/api_clients/certificates.py

```diff
@@ -0,0 +1,68 @@
+"""Client module for external certificate's API integration.
+
+Classes:
+    ExternalCertificatesApiClient: Class to interact with NELP external certificates service.
+"""
+from django.conf import settings
+
+from eox_nelp.api_clients import AbstractApiClient
+
+
+class ExternalCertificatesApiClient(AbstractApiClient):
+    """Allow to perform multiple external certificates operations."""
+
+    def __init__(self):
+        client_id = getattr(settings, "EXTERNAL_CERTIFICATES_API_CLIENT_ID")
```

Comment:
This work, if the auth, is made like Django client credentials body way.
If this change, we have to adapt our AbstractApiClient to work like that or add another method to authenticate....
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/1290a153-3427-46af-8fb9-45005ad642e2)
I say this because at this moment we don't know what is the way of auth.
It could be different ways like
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/9732edff-7e1a-4573-8024-88e5ff523296)

Reply:
yes, since we don't know we should implement the standard way